local-devel/Makefile

```diff
@@ -140,6 +140,15 @@ var/log/buildout-install.log: \
 
 var/log/apt-install-local.log: build-deps.txt run-deps.txt
 	mkdir -pv "./$(dir $(@))"
+	which apt-get || (
+	    set +x
+	    echo "ERROR: Automatic installation of OS deps is only supported on Debian/Ubuntu."
+	    echo "       Install equivalents for your OS of the packages in:"
+	    echo "       \`$(^:%=./%)"\`
+	    echo "       Then run:"
+	    echo "       \`$$ touch \"./var/log/apt-install-local.log\"\`"
```

Comment:
Niice, it could also be a skip prompt [yN]

Reply:
I have a slight preference for exiting with a non-zero status when something requires the developer's attention.  They really need to pay attention to this issue and think about it if they're not on Debian/Ubuntu, so I want to avoid the temptation to hit `y` instead of actually reading and thinking (I know I would likely do that).  Also, if a developer uses something that provides notifications on long running command exits, such as [`ntfy`](https://github.com/dschep/ntfy) which I have some to depend on, then having an actual exit is better there too.